### PR TITLE
README stuff

### PR DESCRIPTION
**Pull request by Exerra**
This is a collection of README related changes

### CODE REVIEW

**Comment by Exerra** [comment]
Finished the README instructions. Might move to a doc page later, but for now it will be ok.

**Comment by Exerra** [comment]
The review requests are meant so you read the README and tell me what to change (if it needs to be changed)

**Comment by Exerra** [comment]
Oh and to make it easier to review, go here [https://github.com/Exerra-Discord/karen-bot/tree/README-stuff](https://github.com/Exerra-Discord/karen-bot/tree/README-stuff) and read the README

File: README.md [hunk 1]
```diff
@@ -32,6 +34,77 @@ TODO: finish readme later
 * Keep commit titles short and then explain them in the description of the commit.
 * **DO NOT PUSH TO MASTER BRANCH!** Push to a new branch (e.g username-whatyouchanged and create a PR)
 
+## Setup instructions
+So, Karen Bot is quite tightly integrated with my *private* API, which is good & bad news. Good news is that a lot of strain is taken off of Karen and put on my API, while the bad news is that someone without API access can't really run Karen that good. Fortunately, I have designed a lot of the API to have as little server side processing as possible, ensuring a ~~tedious~~ simple transition to your own API or just saving files on your computer!
+So this section will walk you through the necessary things to get Karen Bot atleast slightly functional :)
```
Comment (Luki120) [inline README.md]: Typo here, separate the "atleast"

File: README.md [hunk 2]
```diff
@@ -32,6 +34,77 @@ TODO: finish readme later
...
+Now you have a shiny Spotify application, great! So next, click on "Show client secret". Make sure not to leak the secret since it is, well, a secret!
+![How to get OAuth token](https://cdn.exerra.xyz/files/png/tutorials/spotify_oauth/get_token.png)
+
+Oh hey, already at the end of this section! Good job, go pat yourself on the back 🥳
+
+### Genius OAuth token
+First, go to [genius.com/developers/](https://genius.com/developers/). Then create on the biggest button you can find, aka "Create an API client"
+![How to navigate the startpage](https://cdn.exerra.xyz/files/png/tutorials/genius_oauth/startpage.png)
+
+Then log in/sign up and you will be presented the API client creation screen. Fill in the name and if you want, the rest of the fields, then click "Save"
```
Comment (Luki120) [inline README.md:72]: I'd put here "you will be presented **with..**"

**Comment by Luki120** [comment]
Check 2 comments and this has my approval Queen! :SGuraAyy:

File: README.md [hunk 3]
```diff
@@ -80,7 +80,7 @@ HOORAY! YOU HAVE DEMONSTRATED AN EXCEPTIONAL ABILITY TO FOLLOW SIMPLE INSTRUCTIO
 ### Clarifai AI Key
 Now now now, this is the last one (for now)! Pat yourself on the back before proceeding.
 
-You done? Good. First, go to [link](link) and sign up
+You done? Good. First, go to [clarifai.com](https://clarifai.com) and sign up
 ![How to sign up](https://cdn.exerra.xyz/files/png/tutorials/clarifai_key/homepage.png)
```
Comment (Exerra) [inline README.md:84]: test

**Review by Exerra — COMMENTED** [review]
just a test

**Review by Luki120 — COMMENTED** [review]
There's no such button aaa, idk, maybe it's this

File: README.md (hunk 2 above)
Comment (Exerra) [reply README.md:72]: Hmm, I think my version is a bit better
Idk how to describe but just imo my way feels "better"

File: README.md (hunk 1 above)
Comment (Exerra) [reply README.md]: Ohh yes yes
I always say "atleast" not "at least" for some reason haha

Comment (Exerra) [reply README.md]: Oke, fixed in commit 6624f04

File: README.md (hunk 2 above)
Comment (Luki120) [reply README.md:72]: Smh, haha okie

**Pull request merged**